_data/harnesses/kafka-producer-application/kafka.yml

```diff
@@ -0,0 +1,166 @@
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/init.adoc
+
+    - title: Get Confluent Platform
+      content:
+        - action: make_file
+          file: docker-compose.yml
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/make-docker-compose.adoc
+
+        - action: execute_async
+          file: tutorial-steps/dev/docker-compose-up.sh
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/start-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/wait-for-containers.sh
+          render:
+            skip: true
+
+    - title: Create a topic
+      content:
+        - action: execute
+          file: tutorial-steps/dev/harness-create-topic.sh
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/create-topic.adoc
+
+    - title: Configure the project
+      content:
+        - action: make_file
+          file: build.gradle
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/make-build-file.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/gradle-wrapper.sh
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/make-gradle-wrapper.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/make-configuration-dir.sh
+          render:
+            file: tutorials/kafka-producer-application/kafka/markup/dev/make-config-dir.adoc
+
+        - action: make_file
```

Comment:
FYC: this should be a separate `title` because it's not for the project, it's about producer properties?